### DISCUSSION TITLE
Question: Pulse relay

### DISCUSSION THREAD

**Discussion by BruceRU** [arendst/Tasmota#10425] [Q&A] [answered]
Ability to configure the output relay power when "On" or "Off" commands are sent.

- Set "On" or "Off" times (i.e .5 - 60 secs), returning the  status to the pre command state.
- Set the number of  pulses (i.e 1 to 10).
- SOS mode (3 long, 3 short, 3 long, pause, repeat until off cmd is sent

**Comment by ascillato** [comment, marked as answer]
Please read the docs at commands and look for the PULSETIME command. Thanks.

**Reply by BruceRU** [reply]
Thank you for that.
I now see the docs at:
https://tasmota.github.io/docs/Commands/#power

This is very interesting.  I should have known this was already in place.

**Comment by sfromis** [comment]
To have a number of pulses, you may also want to check BlinkTime and BlinkCount, and the blink option of the Power command.

**Reply by BruceRU** [reply]
Please excuse my Nubeness.  I'm sure I'm missing something here.

After playing with the PulseTime, BlinkTime/BlinkCount commands, it seems to operate erratically and inconsistently as far a timing goes.
Are there adjustments for off time between on times?
Is it even possible to get consistent 1 sec (+/-10%) on & 1 sec off pulse timing?

Are the PulseTime{x} command duty-cycles adjustable somehow?

I've tried using the Backlog commands, with the same results.  
All of this has been done thru the Tasmota console.

I wouldn't think so, buy is the output relay timing somehow dependent on network bandwidth?

TIA

**Reply by sfromis** [reply]
You should not try to mix PulseTime (which has no concept of repeats or off time) and blink (which allows repeated on/off cycles). Both should be reasonably good on timing, as long as it is not very brief cycles. Network bandwidth should not be an issue, but I'm not sure what could happen if the very low bandwidth needs are not there.

**Comment by meingraham** [comment]
Your use case may require use of rules and rules timers... but in any case, I would recommend that you move this discussion to Discord and that is a more interactive channel and will be better able to assist you.

Please close this issue and address this to the [Tasmota Support Discord Chat](https://discord.gg/Ks2Kzd4).

Please check the [Contributing Guideline and Policy](https://github.com/arendst/Sonoff-Tasmota/blob/development/CONTRIBUTING.md) and the [Support Guide](https://github.com/arendst/Sonoff-Tasmota/blob/development/SUPPORT.md).

Thanks.

### Support Information
See [Wiki](https://tasmota.github.io/docs/#/Home) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Community](https://groups.google.com/d/forum/sonoffusers) for forum.
See [Code of Conduct](https://github.com/arendst/Sonoff-Tasmota/blob/development/CODE_OF_CONDUCT.md)
